### ISSUE TITLE
NTR: Conjunctival biopsy: spheroids (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9239]
New term request Conjunctival biopsy: spheroids (segal_110421155504):

**Comment by pnrobinson** [comment]
@MickeySegal
please check whether this refers to this finding: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9358195/

**Comment by MickeySegal** [comment]
They don't mention axons.  Our finding is based on https://www.ncbi.nlm.nih.gov/books/NBK1675/ "If no PLA2G6 pathogenic variants are identified but the evolving phenotype remains most consistent with INAD or atypical NAD, a biopsy for identification of dystrophic axons (axonal spheroids) can be considered. Electron microscopic (EM) examination of nerve ultrastructure should be done on one of the following preferred tissues: conjunctiva, skin, rectum, muscle, or other peripheral nerve (sural)."

**Comment by pnrobinson** [comment]
@MickeySegal 
For conjunctival spheroids, this is what I am finding
https://pubmed.ncbi.nlm.nih.gov/32076391/
I am wondering if this is not well defined enough to make a term at this time?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742877/ defines "Axonal spheroids are bubble-like biological features that form on most degenerating axons".

**Comment by pnrobinson** [comment]
@MickeySegal  -- but the above is axonal spheroids and this term request is for conjunctival spheroids?

**Comment by MickeySegal** [comment]
As quoted above from https://www.ncbi.nlm.nih.gov/books/NBK1675/ , axonal spheroids can be detected in a conjunctival biopsy.

**Comment by pnrobinson** [comment]
@MickeySegal 
ok, adding as a child of abn. neuron morphology since this is not a conjunctiva specific finding!
Adding new term: 
- ID: HP:6000717
- label: Axonal spheroids
- parents: [Abnormal neuron morphology (HP:0012757)](https://hpo.jax.org/app/browse/term/HP:0012757)
- pmid: PMID:20301718|PMID:34678457
- def: During neurodegeneration, bubble-like swellings form along the length of the axon, a primary and early effect of dynamic axonal deformation. These varicosities, spheroids, or swellings have been described as beads on a string and branches bearing fruit, and they frequently appear on degenerating axons.
- comment: Axonal spheroids have been found in developmental degeneration and the aging brain, as well as in injury, neuropathy, neuroaxonal dystrophy, and many neurodegenerative diseases. Axonal spheroids can be observed in neurons in a conjunctival biopsy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9239
- synonyms:
